README.md

```diff
@@ -26,6 +26,13 @@ Regardless of whether that source is a relational database like ours, or it's li
 
 This means an user familiar with SQL will immediately feel at home when accessing remote data through Virtual Schemas.
 
+Other databases vendors offer similar features and refer to them as `External Tables` or `Foreign Data Wrapper (FDW)`.
+
+This repository contains Virtual Schema adapters for different external data sources.
+Each of these adapters translates between the Exasol SQL dialect and the SQL dialect of the external data source.
```

Comment:
These two sentences look redundant to me.  The line 23 has the same information

Reply:
I tried to introduce the ther `Virtual Schema adapter` here. To make a difference between `Virtual schema `and `Virtual Schema adapter`